package.nix

```diff
@@ -23,5 +23,7 @@ pkgs.rustPlatform.buildRustPackage {
 
   postInstall = ''
     wrapProgram $out/bin/devenv --set DEVENV_NIX ${inputs.nix.packages.${pkgs.stdenv.system}.nix} --prefix PATH ":" "$out/bin:${inputs.cachix.packages.${pkgs.stdenv.system}.cachix}/bin"
+    # TODO: problematic for our library...
```

Comment:
We could use [`env!`](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env) to embed a default path to nix in the library. That would remove the runtime requirement.